locale/en/black-lives-matter.md

```diff
@@ -0,0 +1,204 @@
+* Donate to the [Justice for Mike Ramos](https://www.gofundme.com/f/justice-for-mike-ramos) fundraiser.
+* Find and donate to bail funds via the [National Bail Fund Network](https://www.communityjusticeexchange.org/nbfn-directory).
+* [Reclaim the Block](https://www.facebook.com/reclaimtheblock/) has published a [Google Doc of organizations to donate to](https://docs.google.com/document/d/1yLWGTQIe3967hdc9RSxBq5s6KKZHe-3_mWp5oemd7OA/preview?pru=AAABcpUiX3k*Y6Q4I6UBtkH3lLz9GVLg0A).
+
+<section id="update" class="center">
+
+*We will update this list as appropriate*
+
+</section>
+
+## Our Commitment
+
+It would be wrong for us to ignore the unearned privilege that exists in
+Node.js. Much of the project leadership is white, and a majority are men.
+We are and have been systematically complicit in perpetuating the issues
+that led us to where we are. We can't change how we've built out the
+project's leadership to date, but we can change how we build it moving
+forward.
+
+We are dedicated to being a part of the solution. We will actively center
+the Black community and uplift Black voices. We will share projects
+founded and built by Black community members. We will hold ourselves
+accountable, fulfill our promises, and make changes as needed. While it is
+our responsibility to hold ourselves accountable, we welcome your help. We
+invite you to share ways that we can do better. Email
+[blacklivesmatter@nodejs.org](mailto:blacklivesmatter@nodejs.org)
+or reach out to any of our project leaders.
+
+<section id="here-for-you">
+
+To Black folks looking for access: We're here for you.
+
+To Black people looking for amplification: We're here for you.
+
+To Black members of our community seeking collaboration: We're here for
+you.
+
+</section>
+
+The responsibility to dismantle white supremacy does not fall on the Black
+community. It falls on the rest of us.
+
+## Taking Action
+
+We will work with any Black community member who wishes to contribute to
+Node.js. Please contact us at [blacklivesmatter@nodejs.org](mailto:blacklivesmatter@nodejs.org)
+or [@nodejs](https://twitter.com/nodejs) on Twitter with any questions.
+
+We have an upcoming free online [collaboration summit](https://github.com/openjs-foundation/summit#upcoming-events).
+There are [many different sessions](https://github.com/openjs-foundation/summit/issues?q=is%3Aissue+is%3Aopen+label%3A%22Session+Proposal%22)
+across various [OpenJS projects](https://openjsf.org/projects/), including Node.js.
+We have an [orientation guide](https://github.com/openjs-foundation/summit/blob/master/2020-06-23-Austin/orientation.md)
+to help you figure out how you might want to take part. If you have any questions or
+want to talk about anything, please reach out to [blacklivesmatter@nodejs.org](mailto:blacklivesmatter@nodejs.org).
+
+Moving forward we plan to offer ongoing support, mentorship, and sponsorship to
+
```

Comment:
```suggestion
```

Reply:
this will be fixed in https://github.com/nodejs/nodejs.org/pull/3254